.github/workflows/upgrade-tests.yaml

```diff
@@ -0,0 +1,89 @@
+name: Upgrade Tests
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  upgrade-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+      cancel-in-progress: true
+
+    env:
+      BRANCH_NAME: ${{ github.event.inputs.branch || 'master' }}
+
+    steps:
+      - name: Checkout noobaa-core (master/input branch)
+        uses: actions/checkout@v4
+        with:
+          repository: 'noobaa/noobaa-core'
+          path: 'noobaa-core'
+          ref: ${{ env.BRANCH_NAME }}
+
+      - name: Deploy minikube
+        run: |
+          cd ./noobaa-core
+          sudo bash ./.travis/deploy_minikube.sh
+
+      - name: Build noobaa
+        run: |
+          cd ./noobaa-core
+          make noobaa NOOBAA_TAG=noobaa-core:${{ env.BRANCH_NAME || 'master' }}
+
+      - name: Checkout noobaa-core (latest)
+        uses: actions/checkout@v4
+        with:
+          repository: 'noobaa/noobaa-core'
+          path: 'noobaa-core'
+
+      - name: Build noobaa (latest)
+        run: |
+          cd ./noobaa-core
+          make noobaa NOOBAA_TAG=noobaa-core:upgrade-tests
+
+      - name: Checkout noobaa-operator
+        uses: actions/checkout@v4
+        with:
+          repository: 'noobaa/noobaa-operator'
+          path: 'noobaa-operator'
+
+      - name: Change settings for k8s and minikube
+        run: |
+          sudo mv /root/.kube /root/.minikube $HOME
+          sudo chown -R $USER $HOME/.kube $HOME/.minikube
+          sed "s/root/home\/$USER/g" $HOME/.kube/config > tmp; mv tmp $HOME/.kube/config
+
+      - name: Build operator
+        run: |
+          cd ./noobaa-operator
+          make all
+
+      - name: Install noobaa system (from master)
+        run: |
+          cd ./noobaa-operator
+          ./build/_output/bin/noobaa-operator crd create
+          ./build/_output/bin/noobaa-operator operator install
+          ./build/_output/bin/noobaa-operator system create  \
+          --db-resources='{ "limits": {"cpu": "200m","memory": "2G"}, "requests": {"cpu": "200m","memory": "2G"}}' \
+          --core-resources='{ "limits": {"cpu": "200m","memory": "1G"}, "requests": {"cpu": "200m","memory": "1G"}}' \
+          --endpoint-resources='{ "limits": {"cpu": "200m","memory": "1G"}, "requests": {"cpu": "200m","memory": "1G"}}' \
+          --noobaa-image='noobaa-core:${{ env.BRANCH_NAME || 'master' }}'
+          ./build/_output/bin/noobaa-operator status
+
+      - name: Wait for phase Ready in the backingstore pod
+        run: |
+          cd ./noobaa-operator
+          ./.travis/number_of_pods_in_system.sh --pods 5
```

Comment:
I'm not sure we still need this.
In PR #7237 I mentioned that:

> Run a script to wait until we have the backingstore pod (in k8s we cannot wait for a resource that doesn't exist (more information https://github.com/kubernetes/kubectl/issues/1516, based on the change we did in https://github.com/noobaa/noobaa-operator/pull/1063).

But it was on March 2023 and since that time I think the issue was closed with a solution that we can use.
Anyway, in version 4.19 the number of pods would probably change, so better avoid it if we can.